### ISSUE TITLE
The training params for ResNet38_aff

### ISSUE THREAD

**Issue by YudeWang** [jiwoon-ahn/psa#18]
@jiwoon-ahn hi,
Thanks for your nice work! I load your shared weights and achieve the same mIoU as you showed in README. 

While I meet some trouble to train the model by myself and only achieve 59.077% mIoU (instead of 60.2%) on val set by ResNet38_aff. As you mentioned in other issues that the default params setting is just for vgg16, could you share the params setting for resnet38_aff?

Here is my res38_aff training setttings:
lr=0.01
gpu=4 (may cause different batch number on each gpu)
batchsize=8
max_epoch=8
loss_weight=1/4 * bg + 1/4 * fg + 1/2 * neg
pretrained_model=res38_cls.pth (provided by you)
alpha=4/16/32
other default params...

**Comment by Seohyeong** [comment]
I'm having difficulty training AffinityNet (corresponding to step 3. Train AffinityNet with the labels and 4. Perform Random Walks on CAMs), getting mIoU below 50%. Could you tell me which weights you used getting 59.077% mIoU on validation set? Did you use res38_cls.pth(output of step1) to do step 3 and used produced resnet38_aff (output of step 3) as weights for step 4?

**Comment by YudeWang** [comment]
@SeoHyeong 
I remember I use imagenet pretrained resnet38 weights and I achieve the performance in his README table with 2 gpus instead of 4.

**Comment by Seohyeong** [comment]
@YudeWang Thanks for your reply! I'll give it a try again with the pretrained weights.

**Comment by yuleung** [comment]
@YudeWang I have a question that what does the alpha=4/16/32 mean?  
In ResNet38_aff, alpha was be set 8?   I'm confused about it.
So, Could you please tell me the reason？ Very grateful to you！
